composer.json

```diff
@@ -18,6 +18,7 @@
     "require": {
         "php": ">=7.1",
         "ext-json": "*",
+        "ext-posix": "*",
```

Comment:
This makes BLT not work on Windows because there's no posix extension on windows. Is the posix extension really necessary? BLT only uses that to determine the home and user directories which on posix systems are in the environment anyway, USER and HOME.

Reply:
Thanks, it looks like you are right and `HOME` is [part of the Posix spec](http://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap08.html#tag_08_03). Obviously we can't control if people screw up their environment variables and throw off BLT, but this doesn't seem any more or less brittle than using the posix extensions.

Reply:
The posix calls were added in https://github.com/acquia/blt/pull/1534 and I don't see any indication there of why they were used instead of a simple check of `$HOME`.